Watchman.Discord/Areas/Help/Services/HelpDBGeneratorService.cs

```diff
@@ -24,14 +26,26 @@ public void FillDatabase(IEnumerable<CommandInfo> commandInfosFromAssembly)
             var helpInfosInDb = _session.Get<HelpInformation>().ToList();
 
             var newCommands = FindNewCommands(commandInfosFromAssemblyList, helpInfosInDb).ToList();
-            
-            newCommands.ForEach(x => _session.Add(this._helpInformationFactory.Create(x)));
+            Task.Run(() => CheckIfExistsUselessHelp(commandInfosFromAssemblyList, helpInfosInDb));
+
+            var newHelpInfos = newCommands.Select(x => _helpInformationFactory.Create(x));
+            _session.Add(newHelpInfos);
         }
 
-        private IEnumerable<CommandInfo> FindNewCommands(IEnumerable<CommandInfo> commandInfosFromAssembly, IEnumerable<HelpInformation> helpInfosInDb)
+        private IEnumerable<CommandInfo> FindNewCommands(IEnumerable<CommandInfo> commandInfosFromAssembly, IEnumerable<HelpInformation> helpInfosInBase)
         {
-            var defaultHelpInfosInDb = helpInfosInDb.Where(x => x.IsDefault).ToList(); // for optimize checking only defaults
+            var defaultHelpInfosInDb = helpInfosInBase.Where(x => x.IsDefault).ToList(); // for optimize checking only defaults
```

Comment:
myśle że samo helpInfos albo helpInformations wystarczy 
to InBase jest zbędne 
plus z tego poziomu kodu nie wiadomo czy dane są w bazie danych 
pod spodem może być implementacja zapisywania do innego miejsca